### PR TITLE
feat: initial implementation

### PR DESCRIPTION
**Pull request by smira**
The whole package is go generated from `pci.ids` into static Go
`map`s.

Signed-off-by: Andrey Smirnov <andrey.smirnov@talos-systems.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# syntax = docker/dockerfile-upstream:1.2.0-labs
+
+# THIS FILE WAS AUTOMATICALLY GENERATED, PLEASE DO NOT EDIT.
+#
+# Generated on 2022-05-20T16:42:27Z by kres 0bf4e28-dirty.
+
+ARG TOOLCHAIN
+
+# runs markdownlint
+FROM node:14.8.0-alpine AS lint-markdown
```
Comment (frezbo) [inline Dockerfile:10]: okay, we need to update this in kres :sweat_smile: , 18.2.0 is latest

Comment (smira) [reply Dockerfile:10]: yep, please PR that! :)

Comment (smira) [reply Dockerfile:10]: or we could enable dependabot on Kres repo, so that we know we need to update Kres, because Kres build instructions are outdated

**Review by frezbo — APPROVED** [review]

**Pull request merged**